src/types/dataModel/backgroundCheckWebhookPayload.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+const zBackgroundCheckWebhookPayload = z.object({
+  data: z.object({
+    employee_email: z.string(),
+    overall_status: z.string(),
+  }),
+  event_type: z.string(),
```

Comment:
also can you change this to be `z.literal("overall_status_update")`. I realized that springverify may add more webhooks in the future so we want to make sure we are only handling the overall status update ones.